src/SFML/Graphics/Font.cpp

```diff
@@ -136,6 +136,9 @@ m_info(copy.m_info),
 m_pages(copy.m_pages),
 m_pixelBuffer(copy.m_pixelBuffer)
 {
+#ifdef SFML_SYSTEM_ANDROID
+    m_stream = copy.m_stream;
+#endif
```

Comment:
I guess we can just implement copy with `= default` now.

Reply:
Yeah that does seem to be the case. It makes me wonder why it wasn't always defaulted to begin with. Is there some intent because this hand-written copy constructor that we're not recognizing?

Reply:
The reason was it was actually not copyable when a `std::unique_ptr` was used. See https://github.com/SFML/SFML/pull/2355#discussion_r1067644147